orderbook/src/api/handler.rs

```diff
@@ -83,6 +84,20 @@ pub async fn get_orders(orderbook: Arc<OrderBook>) -> Result<impl warp::Reply, I
     Ok(with_status(json(&orders), StatusCode::OK))
 }
 
+pub async fn get_order_by_uid(
+    uid: OrderUid,
+    orderbook: Arc<OrderBook>,
+) -> Result<impl warp::Reply, Infallible> {
+    if let Some(order) = orderbook.get_order_by_uid(uid).await {
+        Ok(with_status(json(&order), StatusCode::OK))
+    } else {
+        Ok(with_status(
+            json(&json!({ "description": "Order was not found"})),
```

Comment:
ubernit: I have no particular preference, but let's try to unify the response type when we are just returning a single error message (either as just a string as [here](https://github.com/gnosis/gp-v2-services/pull/111/files#diff-9ebbf7bf2a6a172779327578c2fc9dee475881fdbb9d4e326bf78c0bbaa7a522R86), or an object with one description field).

Reply:
Reading https://github.com/gnosis/gp-v2-services/issues/115 I guess this format here is better since consistent for the frontend.

Reply:
It's probably a good idea to make what is now "OrderPostError" just "ApiError" and return this as json for every error. That ensures that any error is always the same json format. Don't have to do it in this PR.